apps/client/src/app/pages/map/LoginPage.tsx

```diff
@@ -1,25 +1,63 @@
 import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+import axios, { AxiosResponse } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 import TextBoxEmail from './components/login/TextBoxEmail';
 import TextBoxPassword from './components/login/TextBoxPassword';
 import handleLogin from './hooks/useLogin';
 import Button from './shared-ui-components/Button';
 
 function LoginPage() {
+  const [isCredentialValid, setIsCredentialValid] = useState(false);
+  const [isCredentialError, setIsCredentialError] = useState(true);
+  const navigate = useNavigate();
+  const userCredentialHandle = async (
+    email: string,
+    password: string
+  ): Promise<void> => {
+    let postRequest: AxiosResponse;
+    try {
+      postRequest = await axios.post('/api/customer', { email, password });
+      setIsCredentialValid(Boolean(postRequest.data));
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('POST request failed:', error);
```

Comment:
Please don't remove or disable any rule of Es Lint, for this line maybe you throw an Error than only console this.